chapters/synchronous.tex

```diff
@@ -405,7 +405,7 @@ \section{Clock Constructors}\label{clock-constructors}
 The optional $\mathit{startInterval}$ argument (defaults to 0) is the value returned by \lstinline!interval()! at the first tick of the clock, see \cref{initialization-of-clocked-partitions}.
 The result is of base type \lstinline!Clock! that ticks when \lstinline!edge($\mathit{condition}$)! becomes \lstinline!true!.
 \begin{nonnormative}
-This clock is used to trigger a clocked partition due to a state event, that is a zero-crossing of a \lstinline!Real! variable, in a continuous-time partition or due to a hardware interrupt that is modeled as \lstinline!Boolean! in the simulation model.
+This clock is used to trigger a clocked base-clock partition due to a state event, that is a zero-crossing of a \lstinline!Real! variable, in a continuous-time base-clock partition or due to a hardware interrupt that is modeled as \lstinline!Boolean! in the simulation model.
```

Comment:
The placement of commas makes this sentence hard to parse (not really the topic of this PR, but it makes it easier to accept the new text):
```suggestion
This clock is used to trigger a clocked base-clock partition due to a state event (that is a zero-crossing of a \lstinline!Real! variable) in a continuous-time base-clock partition, or due to a hardware interrupt that is modeled as \lstinline!Boolean! in the simulation model.
```